frontend/src/layouts/JiraDashboard/useEmployeeFilter.js

```diff
@@ -0,0 +1,22 @@
+import { useState, useEffect } from "react";
+import { useFetchEmployeeTeamData } from "../Team/teamHooks";
+
+const useEmployeeFilter = () => {
+  const [loading, data, error] = useFetchEmployeeTeamData();
+  const [employeeDetails, setEmployeeDetails] = useState([]);
```

Comment:
why is employeeDetails state needed? It is just a derived version of data. no need to keep it in state